src/routes/index.js

```diff
@@ -23,7 +23,13 @@ export default function getRoutes(renderRequestMap, apiProxyFn$ = apiProxy$) {
 		method: 'GET',
 		path: '/_ah/{param*}',
 		config: { auth: false },
-		handler: (request, reply) => reply('OK'),
+		handler: (request, reply) => {
+			if (request.params.checkType === 'error') {
```

Comment:
I think this would be `request.params.param === 'error'` if you want to test `/_ah/error` or `request.query.checkType === 'error'` if you want to test `/_ah/?checkType=error`